plugins/hydra_rq_launcher/README.md

```diff
@@ -0,0 +1,67 @@
+# Hydra RQ Launcher
```

Comment:
follow the structure with the other plugins.
1. README is nearly empty and has a link to a website page.
2. For the website page see what other similar plugins looks like.
(header, structure etc).